spec/swagger.yaml

```diff
@@ -6314,23 +6322,14 @@ paths:
         401: *Error401ResponseDefinition
         500: *Error500ResponseDefinition
     post:
-      summary: Execute Batch
+      summary: Execute batch
       description: |-
-        Triggers asynchronous executing Batch.
-        In the Execute flow Batch waits until all files are uploaded and attached or failed on one of these operations.
-        When all these asynchronous processes are completed job `authorize` is triggered if was requested and Batch is turned into the `COMPLETED` status
+        Triggers asynchronous execution of a batch.
+        
+        In the execution flow, the batch waits until all files are uploaded and attached, or fails on one of those operations. When all of these asynchronous processes are completed, job `authorize` is triggered if it was requested, and the batch becomes a status of `COMPLETED`.
```

Comment:
> the batch waits ..., or fails on one of those operations.   

Yeah, the original sentence was quite complicated :)

The batch doesn't fail, it waits until upload and attach processes are completed successfully or failed for any reason.

Reply:
Fixed.